backend/src/packages/meditation/meditation.service.ts

```diff
@@ -0,0 +1,49 @@
+import { type Service } from '#libs/types/types.js';
+
+import {
+  type MeditationEntryRequestDto,
+  type MeditationEntryResponseDto,
+} from './libs/types/types.js';
+import { MeditationEntity } from './meditation.entity.js';
+import { type MeditationRepository } from './meditation.repository.js';
+
+class MeditationService implements Service {
+  private meditationRepository: MeditationRepository;
+
+  public constructor(meditationRepository: MeditationRepository) {
+    this.meditationRepository = meditationRepository;
+  }
+
+  public find(): ReturnType<Service['find']> {
+    return Promise.resolve(null);
+  }
+
+  public async findAll(): ReturnType<Service['findAll']> {
+    return await Promise.resolve({ items: [] });
+  }
+
+  public async create({
+    topicName,
```

Comment:
let's not pass topicId or name at all for now
```suggestion
```

Reply:
Seems to be done. I leave request dto with topicName, should I delete it as well?